app/sender/messageSender.go

```diff
@@ -37,14 +37,23 @@ func SendMessageHelper(to, message, file string) (error, *store.Message) {
 		m.ExpireTimer = session.ExpireTimer
 		_, savedM := store.SaveMessage(m)
 
-		go SendMessage(session, m)
+		go func() {
+			mNew, err := SendMessage(session, m)
+			if err != nil {
+				// log.Errorln("[axolotl] Error sending message:" + err.Error())
```

Comment:
This should be uncommented I guess?

Reply:
we have already a error message down the stack, we don't need to log it twice, so i removed it.